### PR TITLE
Adds deprecation warnings.

### PR DESCRIPTION
**Pull request by dbernstein**
fcrepo-audit will be discontinued with the  5.0 release.

Resolves: https://jira.duraspace.org/browse/FCREPO-2660

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,5 @@ fcrepo-audit
 [![Build Status](https://travis-ci.org/fcrepo4-exts/fcrepo-audit.png?branch=master)](https://travis-ci.org/fcrepo4-exts/fcrepo-audit)
 
 
-Provides audit functionality for the Fedora Commons repository framework. The [fcrepo-webapp-plus](https://github.com/fcrepo4-exts/fcrepo-webapp-plus) project provides a convenient option to build fcrepo-webapp packaged with fcrepo-audit module.
\ No newline at end of file
+Provides audit functionality for the Fedora Commons repository framework. The [fcrepo-webapp-plus](https://github.com/fcrepo4-exts/fcrepo-webapp-plus) project provides a convenient option to build fcrepo-webapp packaged with fcrepo-audit module.
+This functionality has been deprecated as of v.4.7.5.  It will no longer continue to be supported by 5.x.
```
Comment (bseeger) [inline README.md]: I think my preference would be to have this line  as the very first thing on this page under build status.  Maybe in bold as well, so it stands out.

**Comment by bseeger** [comment]
Over all this looks good to me.

File: README.md (hunk 1 above)
Comment (awoods) [reply README.md]: @bseeger : agreed

**Comment by awoods** [comment]
@dbernstein : as a point of practice, the author of a PR is discouraged from merging it.

**Comment by escowles** [comment]
@awoods @dbernstein Though this has evolved a bit since Github added the approval function.  Many projects customarily have reviewers approve PRs and consider it fine for the author to merge them once approved.  E.g., I do this all the time, so I can review before CI finishes.

**Comment by awoods** [comment]
@escowles : I would prefer to eliminate any chance of ambiguity by holding firm on the policy that authors never commit their own PRs.
...but if there is committer consensus for a more lenient policy, I can reevaluate.

**Pull request merged**